ivy/functional/frontends/jax/numpy/mathematical_functions.py

```diff
@@ -425,6 +425,11 @@ def floor_divide(x1, x2, /, out=None):
 
 
 @to_ivy_arrays_and_back
+def prod(x, /, out=None):
+    return ivy.prod(x, out=out)
+
+
 def inner(a, b):
```

Comment:
Please return the initial `@to_ivy_arrays_and_back` decorator back to the `inner` function. You don't need to change it
```suggestion

@to_ivy_arrays_and_back
def inner(a, b):
```

Reply:
Sorry I might have accidentally removed the decorator. Brought it back and set the data type for the jax numpy prod function.